src/OfficeService.Models.Db/DbOffice.cs

```diff
@@ -47,6 +51,14 @@ public void Configure(EntityTypeBuilder<DbOffice> builder)
         .Property(o => o.Address)
         .IsRequired();
 
+      builder
+        .Property(o => o.Latitude)
```

Comment:
is it necessary to use IsRequired() here? Latitude and Longitude are double, double can't be null

Reply:
yes, you right